packages/apps/src/index.tsx

```diff
@@ -17,10 +17,11 @@ import { BlockAuthors, Events } from '@polkadot/react-query';
 import settings from '@polkadot/ui-settings';
 
 import Apps from './Apps';
+import theme from '../../../styled-theming';
```

Comment:
the imports are absolute in the rest of apps.. can we make this absolute too?
import theme from '@polkadot/styled-theming';

Reply:
We seems not to publish it in polkadot packages, and we had a long discussion on where to put the lib, could be either in a separated github account, or under cennznet, but left the decisions to leads.
For now, it we want always use absolute path, could try it by implementing 'babel-plugin-module-resolver' like:

```js
require('babel-plugin-module-resolver'),
        {
            'styled-theming': './styled-theming',
          },
        },
```

Thats why I used relative paths for now, any inputs @holygits

Reply:
I don't see a good reason to separate this package from the UI at this stage so IDM putting under `packages/` or leave relative path which ever is simplest

Reply:
👌